.github/CONTRIBUTING.md

```diff
@@ -54,16 +54,20 @@ Once the pull request is approved, a team member will take care of merging.
 
 ## Setup environment
 
-Setting up your KerasCV development environment requires you to fork the KerasCV repository,
-clone the repository, install dependencies, and execute `python setup.py develop`.
+Python 3.7 or later is required.
+
+Setting up your KerasCV development environment requires you to fork the
+KerasCV repository, clone the repository, create a virtual environment, and 
+install dependencies.
 
 You can achieve this by running the following commands:
 
 ```shell
 gh repo fork keras-team/keras-cv --clone --remote
 cd keras-cv
-pip install ".[tests]"
-python setup.py develop
+python -m venv ~/keras-cv-venv
```

Comment:
can we preserve the original instructions as well and offer this as an alternative?

Reply:
Sure!